### PR TITLE
Add Polish localization - References

### PR DESCRIPTION
**Pull request by mfilocha**
This is a part of the initial Polish localization - see: https://github.com/kubernetes/website/pull/18419

### CODE REVIEW

File: content/pl/docs/reference/_index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: Materiały źródłowe
+approvers:
+- chenopis
```
Comment (rlenferink) [inline content/pl/docs/reference/_index.md]: Can you remove these lines? We want to use the reviewers/approvers from the OWNERS file instead. Only when the localization will be having a lot of people, files can be appointed to specific contributors.

File: content/pl/docs/reference/glossary/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+approvers:
+- chenopis
+- abiogenesis-now
```
Comment (rlenferink) [inline content/pl/docs/reference/glossary/index.md]: Can you remove these lines? We want to use the reviewers/approvers from the OWNERS file instead. Only when the localization will be having a lot of people, files can be appointed to specific contributors.

File: content/pl/docs/reference/tools.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+reviewers:
+- janetkuo
```
Comment (rlenferink) [inline content/pl/docs/reference/tools.md]: Can you remove these lines? We want to use the reviewers/approvers from the OWNERS file instead. Only when the localization will be having a lot of people, files can be appointed to specific contributors.

**Comment by sftim** [comment]
/hold
pending approval & merge for #18419. Once that merges, it's OK to /unhold
People can /lgtm (for localization review) and /approve, even whilst this is held.

File: content/pl/docs/reference/_index.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  * [1.14](/docs/reference/generated/kubernetes-api/v1.14/)
+  * [1.13](/docs/reference/generated/kubernetes-api/v1.13/)
+
+## Biblioteki klientów API
+
+Aby wywołać Kubernetes API z wybranego języka programowania, możesz skorzystać z
+[bibliotek klienckich](/docs/reference/using-api/client-libraries/). Oficjalnie wspierane
+biblioteki to:
+
+* [Kubernetes Go client library](https://github.com/kubernetes/client-go/)
```
Comment (nvtkaszpir) [inline content/pl/docs/reference/_index.md]: Are we going to be translating below urls to the letter?

File: content/pl/docs/reference/_index.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+* [kubelet](/docs/admin/kubelet/) - Główny agent działający na każdym węźle. Kubelet pobiera zestaw definicji PodSpecs i gwarantuje, że opisane przez nie kontenery poprawnie działają.
+* [kube-apiserver](/docs/admin/kube-apiserver/) - REST API, które sprawdza poprawność i konfiguruje obiekty API, takie jak pody, serwisy czy kontrolery replikacji.
+* [kube-controller-manager](/docs/admin/kube-controller-manager/) - Proces wykonujący główne pętle sterowania Kubernetes.
+* [kube-proxy](/docs/admin/kube-proxy/) - Przekazuje bezpośrednio dane przepływające w transmisji TCP/UDP lub dystrybuuje ruch TCP/UDP zgodnie ze schematem *round-robin* pomiędzy usługi back-endu.
+* [kube-scheduler](/docs/admin/kube-scheduler/) - Scheduler odpowiada za dostępność, wydajność i zasoby.
+
+## Dokumentacja projektowa
+
+Archiwum dokumentacji projektowej różnych funkcjonalności Kubernetes. Warto zacząć od [Kubernetes Architecture](https://git.k8s.io/community/contributors/design-proposals/architecture/architecture.md) oraz [Kubernetes Design Overview](https://git.k8s.io/community/contributors/design-proposals).
```
Comment (nvtkaszpir) [inline content/pl/docs/reference/_index.md]: Are we going to translate url text in here as well, It points outside of the docs, though, so maybe in polish + note in brackets that this is in English?

File: content/pl/docs/reference/glossary/cluster.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+title: Klaster
+id: cluster
+date: 2019-06-15
+full_link: 
+short_description: >
+   Zestaw maszyn, nazywanych węzłami, na których uruchamiane są aplikacje zarządzane przez Kubernetes. Klaster posiada przynajmniej jeden węzeł roboczy i jeden węzeł typu master.
```
Comment (nvtkaszpir) [inline content/pl/docs/reference/glossary/cluster.md]:
```suggestion
   Zestaw maszyn, nazywanych węzłami, na których uruchamiane są aplikacje zarządzane przez Kubernetes. Klaster posiada przynajmniej jeden węzeł roboczy (*node*)i jeden węzeł typu master (*master node*).
```

File: content/pl/docs/reference/glossary/cluster.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+aka: 
+tags:
+- fundamental
+- operation
+---
+Zestaw maszyn, nazywanych węzłami, na których uruchamiane są aplikacje zarządzane przez Kubernetes. Klaster posiada przynajmniej jeden węzeł roboczy i jeden węzeł typu master.
```
Comment (nvtkaszpir) [inline content/pl/docs/reference/glossary/cluster.md]:
```suggestion
Zestaw maszyn, nazywanych węzłami, na których uruchamiane są aplikacje zarządzane przez Kubernetes. Klaster posiada przynajmniej jeden węzeł roboczy (*node*) i jeden węzeł typu master (*master node*).

File: content/pl/docs/reference/glossary/cluster.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+<!--more-->
+Na węźle (lub węzłach) roboczych rozmieszczane są pody, które są składowymi aplikacji. Węzeł (lub węzły) typu master zarządzają węzłami roboczymi i podami należącymi do klastra. Zwielokrotnione węzły typu master zapewniają większą niezawodność i odporność klastra na awarie.
```
Comment (nvtkaszpir) [inline content/pl/docs/reference/glossary/cluster.md]:
```suggestion
Na węźle (lub węzłach) roboczych rozmieszczane są pody, które są częściami składowymi aplikacji. Węzeł (lub węzły) typu master zarządzają węzłami roboczymi i podami należącymi do klastra. Zwielokrotnione węzły typu master zapewniają większą niezawodność i odporność klastra na awarie.
```

File: content/pl/docs/reference/_index.md (hunk 4 above)
Comment (mfilocha) [reply content/pl/docs/reference/_index.md]: I'd rather not.

**Comment by nvtkaszpir** [comment]
/lgtm
other some things can be left for next milestone.

File: content/pl/docs/reference/glossary/container-runtime.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+- fundamental
+- workload
+---
+ *Container runtime* to oprogramowanie zajmujące się uruchamianiem kontenerów.
+
+<!--more-->
+
+Kubernetes obsługuje różne *container runtimes*: [Docker](http://www.docker.com),
+[containerd](https://containerd.io), [cri-o](https://cri-o.io/),
+[rktlet](https://github.com/kubernetes-incubator/rktlet) oraz każdą implementację
```
Comment (nvtkaszpir) [inline content/pl/docs/reference/glossary/container-runtime.md]:
```suggestion
[rktlet](https://github.com/kubernetes-incubator/rktlet) oraz każdą implementację zodną z

File: content/pl/docs/reference/glossary/etcd.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
+---
+title: etcd
+id: etcd
+date: 2018-04-12
+full_link: /docs/tasks/administer-cluster/configure-upgrade-etcd/
+short_description: >
+  Magazyn typu klucz-wartość *(key value)* zapewniający spójność i wysoką dostępność, używany do przechowywania wszystkich danych o klastrze Kubernetes.
```
Comment (nvtkaszpir) [inline content/pl/docs/reference/glossary/etcd.md]:
```suggestion
  Magazyn typu klucz-wartość *(key/value store)*, zapewniający spójność i wysoką dostępność, używany do przechowywania wszystkich danych o klastrze Kubernetes.

File: content/pl/docs/reference/glossary/etcd.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+aka: 
+tags:
+- architecture
+- storage
+---
+Magazyn typu klucz-wartość *(key value)* zapewniający spójność i wysoką dostępność, używany do przechowywania wszystkich danych o klastrze Kubernetes.
```
Comment (nvtkaszpir) [inline content/pl/docs/reference/glossary/etcd.md]:
```suggestion
Magazyn typu klucz-wartość *(key/value store)*, zapewniający spójność i wysoką dostępność, używany do przechowywania wszystkich danych o klastrze Kubernetes.

File: content/pl/docs/reference/glossary/kube-apiserver.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+ Składnik *master* udostępniający API Kubernetes. Służy jako *front-end* dla warstwy sterowania Kubernetes.
+ Serwer API jest składnikiem
+{{< glossary_tooltip text="warstwy sterowania" term_id="control-plane" >}} Kubernetes, który udostępnia API.
+Server API służy jako front-end warstwy sterowania Kubernetes.
+
+<!--more-->
+
+Podstawowa implementacją serwera API Kubernetes jest [kube-apiserver](/docs/reference/generated/kube-apiserver/).
+kube-apiserver został zaprojektowany w taki sposób, aby móc skalować się horyzontalnie &mdash; to oznacza, że zwiększa swoją wydajność poprzez dodawanie kolejnych instancji.
+Można uruchomić kilka instancji kube-apiserver i rozkładać ruch pomiędzy nie.
```
Comment (nvtkaszpir) [inline content/pl/docs/reference/glossary/kube-apiserver.md]:
```suggestion
Można uruchomić kilka instancji kube-apiserver i rozkładać między nimi ruch od klientów.

**Comment by sftim** [comment]
Thanks @mfilocha & @nvtkaszpir 
/approve
(Once the initial localization lands, there will be a set of approvers for Polish who can self-manage the localization)

If new commits come in, and those changes are good, it's OK to add another /lgtm

**Comment by rlenferink** [comment]
_Approved based on @sftim being an [sig-docs approver](https://github.com/kubernetes/website/commit/756e5976846d7a112a833ffe7e448eb734ce17d6) (however, the `dev-1.17-pl.1` branch is a bit behind master)_

File: content/pl/docs/reference/_index.md (hunk 5 above)
Comment (mfilocha) [reply content/pl/docs/reference/_index.md]: I'd leave it for now - to not mislead readers that they are translated.

**Pull request merged**